indico/htdocs/sass/modules/abstracts/_email_templates.scss

```diff
@@ -55,16 +55,59 @@
     }
 }
 
-.email-template-list {
-    list-style-type: none;
-    padding-left: 0;
-    width: 800px;
+#email-template-manager {
+    max-width: 800px;
+
+    .email-template-container {
+        padding: 0 0 1em 1em;
+
+        .email-template-notifications {
+            width: 800px;
+            max-height: 800px;
+            overflow: auto;
+
+            .email-template-list {
+                list-style-type: none;
+                padding-left: 0;
+                padding-right: 20px;
+
+                .email-template {
+                    margin-bottom: 10px;
+
+                    .email-preview {
+                        display: none;
+
+                        .email-subject {
+                            padding-bottom: 10px;
+                        }
 
-    .email-template {
-        margin-bottom: 10px;
+                        .email-body {
+                            color: $light-black;
+                        }
+                    }
+                }
+                .regform-section-sortable-placeholder {
+                    @extend .i-box;
+                    @include border-all($gray, dashed);
+                    @include box-shadow(none);
+                    background-color: $light-gray;
+                    margin: 25px 0;
```

Comment:
I don't think we want to apply a different margin. It should look like it's a socket for the box, so it should have the same margin. `.i-box-group.vert` should take care of it if you add the `.i-box` class to the placeholder html element.